client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -277,7 +279,10 @@ class ThemeEnhancements extends Component {
 	}
 
 	render() {
-		const { siteIsJetpack, translate } = this.props;
+		const { site, siteIsJetpack, translate } = this.props;
+
+		const jetpackVersion = get( site, 'options.jetpack_version', 0 );
+		const siteHasMinileven = jetpackVersion && versionCompare( jetpackVersion, '8.3-alpha', '<' );
```

Comment:
I wonder if instead of introducing a `get()` and manual retrieval of the Jetpack version here we could just use the `isJetpackMinimumVersion` selector with a `=== false` condition?